sources/Inputs/include/NetworkManager.h

```diff
@@ -72,9 +74,9 @@ class NetworkManager {
   void buildTree();
 
  private:
-  boost::shared_ptr<DYN::DataInterface> interface_;                  ///< data interface
-  std::unordered_map<Node::NodeId, boost::shared_ptr<Node>> nodes_;  ///< nodes representing the node tree
-  std::vector<ProcessNodeCallback> nodesCallbacks_;                  ///< list of callback or nodes
+  boost::shared_ptr<DYN::DataInterface> interface_;      ///< data interface
```

Comment:
why keep boost::shared_ptr here ? if not useless include at the top of the file

Reply:
This is the returned value of the Dynawo API